fapi-client/src/main/scala/com/gu/facia/api/utils/ExternalLinks.scala

```diff
@@ -0,0 +1,32 @@
+package com.gu.facia.api.utils
+
+import java.net.URI
+
+import scala.util.Try
+
+object ExternalLinks {
+  val corsOrigins = List(
```

Comment:
Bad name, they are actually the `CORS` list from `frontend`, but in this context they make no sense.